docs/source/keylime/index.rst

```diff
@@ -0,0 +1,97 @@
+Keylime
+=======
+
+ESI provides a guide to use `keylime`_ outside Ironic for run-time attestation with `IMA`_.
+
+Prerequisites
+-------------
+* A Keylime server machine with keylime registrar and verifier installed.
+* A node to be attested with an active TPM 2.0 and keylime agent installed.
+* A private network for attestation workflow.
+
+Build Keylime Server
+--------------------
+Install Keylime packages:
```

Comment:
This section should probably start with provisioning the node that will act as the Keylime server; and then once provisioning is done, one ssh's into that server and does the rest here.

As an additional note, if that server needs to be connected to the public internet for this step, then a public network could be a requirement. Alternatively, the user might acquire keylime $some_other_way. You can describe that option and this one by noting that one can attach the node to the public network using 'esi node network attach'.